applications/launchpad/gui-react/src/containers/TBotContainer/index.tsx

```diff
@@ -1,12 +1,13 @@
 import TBotPrompt from '../../components/TBot/TBotPrompt'
 import { useAppSelector } from '../../store/hooks'
-import { selectTBotOpen, selectTBotQueue } from '../../store/tbot/selectors'
+import { selectTBotQueue } from '../../store/tbot/selectors'
 
 const TBotContainer = () => {
   const tbotQueue = useAppSelector(selectTBotQueue)
-  const tbotOpen = useAppSelector(selectTBotOpen)
 
-  return <TBotPrompt open={tbotOpen} messages={tbotQueue} floating />
+  return (
+    <TBotPrompt open={tbotQueue.length > 0} messages={tbotQueue} floating />
```

Comment:
it could be done like this or `tbotQueue.length > 0` could've been inside `selectTBotOpen` to hide implementation details - if you left the selector in component, but internal state management changed, you would only change selector, the component would not have to change
(not a request for change, just a general comment)